packages/devtools_app/lib/src/primitives/trees.dart

```diff
@@ -48,26 +46,26 @@ abstract class TreeNode<T extends TreeNode<T>> {
   bool get isSelected => _selected;
   bool _selected = false;
 
-  T get root {
+  T? get root {
```

Comment:
this should not be nullable. If a node does not have a parent, we return `this` as the root (e.g. `this.root => this`), and `this.root` would not be able to be accessed if `this` was null.